template-setup.sh

```diff
@@ -2,5 +2,36 @@
 
 mv example.env .env
 
-# Delete this script after running
-rm -- "$0"
+# Function to prompt for 'y' input
+prompt_confirmation() {
+    local prompt_message="$1"
+    read -p "$prompt_message (y/n): " -n 1 -r
+    echo    # Move to a new line
+    [[ $REPLY =~ ^[Yy]$ ]]
+}
+
+# Check if 'just' command is available
+if ! command -v just &> /dev/null
+then
+    echo "'just' command not found. 🤨"
+
+    # Ask to install 'just'
+    if prompt_confirmation "Do you want to install the 'just' command runner?"
+    then
+        cargo install just
+        echo "'just' has been installed."
+    else
+        echo "Installation of 'just' cancelled. Can't install tools. ❌"
+        exit 1
```

Comment:
```suggestion
        exit 0
```
Don't think it's error if user declined to install just

Reply:
or other tools